core.py

```diff
@@ -1,30 +1,29 @@
 import logging
+
 import meetup.api
-from datetime import datetime, timedelta
+
 from config import target
 from timeutils import meetup_time, meetup_duration
```

Comment:
Can't find anything in PEP8 about blank lines separating imports

Reply:
Imports should be grouped in the following order:

1. standard library imports
2. related third party imports
3. local application/library specific imports

You should put a blank line between each group of imports.

* http://pep8.org/#imports
---

If you use PyCharm, you can just go to `Code > Optimize Imports` to get the same results.